uniffi_bindgen/src/bindings/python/templates/RustBufferStream.py

```diff
@@ -107,7 +107,7 @@ def readString(self):
     # The Object type {{ object_name }}.
 
     def read{{ canonical_type_name }}(self):
-        handle = self.readU64();
+        handle = self._unpack_from(8, ">Q")
```

Comment:
The `readU64` method is only present if the component interface uses the `u64` type.